modules/deploying-a-model.adoc

```diff
@@ -43,11 +43,11 @@ NOTE: The *Model framework* list shows only the frameworks that are supported by
 * *To use a new data connection*
 ... To define a new data connection that your model can access, select *New data connection*.
 ... In the *Name* field, enter a unique name for the data connection.
-... In the *AWS_ACCESS_KEY_ID* field, enter your access key ID for Amazon Web Services (AWS).
-... In the *AWS_SECRET_ACCESS_KEY* field, enter your secret access key for the AWS account you specified.
-... In the *AWS_S3_ENDPOINT* field, enter the endpoint of your AWS S3 storage.
-... In the *AWS_DEFAULT_REGION* field, enter the default region of your AWS account.
-... In the *AWS_S3_BUCKET* field, enter the name of the AWS S3 bucket.
+... In the *Access_key* field, enter your access key ID for Amazon Web Services (AWS).
+... In the *Secret_key* field, enter your secret access key for the AWS account you specified.
```

Comment:
For `Access key` and `Secret key`, I don't think these should be separated by underscores. It looks right in the other file you've updated here :)

Reply:
@jbyrne-redhat Thanks John, just addressed this in the latest commit